### PR TITLE
update Core to 12.9

### PR DESCRIPTION
**Pull request by blagoev**

### CODE REVIEW

**Review by nielsenko — COMMENTED** [review]
Core is updated to f6bbd45cbb6cfb3a2d4212eeba59b4a46bca22d6 which does not match with the v12.9.0 tag which is at b86d2107fec4246b55af24a3b11e00275575f808

**Comment by blagoev** [comment]
@nielsenko updated to the tagged release commit

**Review by nielsenko — APPROVED** [review]
LGTM

File: test/credentials_test.dart [hunk 1]
```diff
@@ -284,12 +284,12 @@ Future<void> main([List<String>? args]) async {
     const String newPassword = "!@#!DQXQWD!223eda";
     final authProvider = EmailPasswordAuthProvider(app);
     await authProvider.registerUser(username, strongPassword);
-    await expectLater(() {
+    await expectLater(() async {
```
Comment (desistefanova) [inline test/credentials_test.dart:287]: @blagoev Why do we need `async` and `await` inside `expectLater`? It is not needed. I have also added them by mistake in some of my old tests, but I think I have to remove them.

Comment (blagoev) [reply test/credentials_test.dart:287]: its not problematic per se. you can do it or not do it.

**Pull request merged**